robot_speed: 1
task_execution_time: 0
collision_threshold: 0.5
battery_threshold: 20
max_charging_time: 3600
max_tasks_in_task_list: 5
epsilon: 0.5
initial_resources: 100

routing: True
dmas_update_rate: 5

max_arrival_interval: 20
priorities: [1, 2, 3]

node_locations: 
  - !!python/tuple [10, 30]
  - !!python/tuple [10, 50]
  - !!python/tuple [10, 70]
  - !!python/tuple [30, 10]
  - !!python/tuple [30, 30]
  - !!python/tuple [30, 50]
  - !!python/tuple [30, 70]
  - !!python/tuple [30, 90]
  - !!python/tuple [60, 10]
  - !!python/tuple [60, 30]
  - !!python/tuple [60, 50]
  - !!python/tuple [60, 70]
  - !!python/tuple [60, 90]
  - !!python/tuple [90, 10]
  - !!python/tuple [90, 50]
  - !!python/tuple [90, 90]

node_names: 
  pos_1:
    - pos_5
  pos_2:
    - pos_6
  pos_3:
    - pos_7
  pos_4:
    - pos_5
    - pos_9
  pos_5:
    - pos_1
    - pos_4
    - pos_6
  pos_6:
    - pos_2
    - pos_5
    - pos_7
    - pos_10
    - pos_11
    - pos_12
  pos_7:
    - pos_3
    - pos_6
    - pos_8
  pos_8:
    - pos_7
    - pos_13
  pos_9:
    - pos_4
    - pos_10
    - pos_14
  pos_10:
    - pos_6
    - pos_9
    - pos_11
  pos_11:
    - pos_6
    - pos_10
    - pos_12
    - pos_15
  pos_12:
    - pos_6
    - pos_11
    - pos_13
  pos_13:
    - pos_8
    - pos_12
    - pos_16
  pos_14:
    - pos_15
    - pos_9
  pos_15:
    - pos_11
    - pos_14
    - pos_16
  pos_16:
    - pos_13
    - pos_15

depot_locations:
    - pos_1
    - pos_2
    - pos_3

fm_rate: 2